src/styles/header.scss

```
@@ -27,9 +39,11 @@ $navbar-dropdown-radius: 0;
   .navbar-end {
     .navbar-item {
       font-family: $family-roboto-condensed;
-      font-size: $font-size-h5;
+      font-size: 16px;
       font-weight: bold;
       text-transform: uppercase;
+      line-height:  24px;
```

Line height should be a fraction of the font-size and never an absolute pixel value. In this case
```
24px / 16px = 1.5
```
so
```suggestion
      line-height:  1.5;
```
should be right property.

---

@panchovm correct me if I'm wrong but in places where text doesn't span multiple lines, line height has no effect and we shouldn't be needing that property imo.